resources/rest-service/cloudify/migrations/versions/272e61bf5f4a_6_3_to_6_4.py

```diff
@@ -85,6 +87,21 @@ def add_config_log_fetch_credentials():
     )
 
 
+def add_config_marketplace_url():
+    op.bulk_insert(
+        config_table,
+        [
+            {
+                'name': 'marketplace_api_url',
+                'value': 'https://marketplace.cloudify.co',
+                'scope': 'rest',
+                'schema': {'type': 'string'},
+                'is_editable': True,
```

Comment:
maybe not editable?  do we plan on running multiple instances of Marketplace?  self-hosted?